app/lib/actions/katello/organization/manifest_import.rb

```diff
@@ -26,6 +26,13 @@ def plan(organization, path, force)
         def humanized_name
           _("Import Manifest")
         end
+
+        def finalize
```

Comment:
this task does not call `plan_self` in its `plan` method, therefore finalize is not triggered, which explains why I don't see the audit being added or refreshed_at being updated. simply adding plan_self to the end of sequence block should fix it

Reply:
Yes, right it doesn't trigger finalize. My mistake that I have tested test cases and I wasn't able to test full flow through UI as on dev setup faced some different issue.
I will fix it asap.